hushline/routes.py

```diff
@@ -154,6 +168,10 @@ def submit_message(username: str) -> Response | str:
 
             captcha_answer = request.form.get("captcha_answer", "")
             if not validate_captcha(captcha_answer):
+                # Encrypt the message and store it in the session
+                session["submit_contact_method"] = encrypt_field(form.contact_method.data)
```

Comment:
I don't love the idea of using the same encryption key for data shared externally as we do for data in the db. Especially since in this instance, users can essentially exercise our encryption library with any chosen plaintext.  At minimum we should probably use a separate key for encrypting this data vs the data in the db.